app/app.go

```diff
@@ -400,6 +400,7 @@ func New(
 		app.DistrKeeper,
 		app.StakingKeeper,
 		distrtypes.ModuleName,
+		cast.ToBool(appOpts.Get("telemetry.enabled")),
```

Comment:
is there a case where `appOpts.Get` could panic here?

Reply:
I think it's good, even if telemetry.enabled doesn't exists it doesn't panic. The `cast.ToBool` actually swallows any errors and returns false. This is the suggestions in the cosmos sdk comments "Note, casting Get calls may not yield
	// the expected types and could result in type assertion errors. It is recommend
	// to either use the cast package or perform manual conversion for safety."